src/server/game/Grids/Notifiers/GridNotifiers.cpp

```diff
@@ -179,6 +158,23 @@ void PlayerRelocationNotifier::Visit(PlayerMapType& m)
     }
 }
 
+void PlayerRelocationNotifier::Visit(CreatureMapType& m)
+{
+    bool relocated_for_ai = (&i_player == i_player.m_seer);
+
+    for (CreatureMapType::iterator iter = m.begin(); iter != m.end(); ++iter)
```

Comment:
can use range loop

Reply:
Missed this one I think, but is there anything to gain from range-based loop? 🤔